src/components/pages/Site/Site.js

```diff
@@ -165,6 +171,11 @@ const SiteForm = ({
           helperText={language.helperText.getLatitude()}
           shouldShowSteps={true}
           step="0.000001"
+          addInputButton={true}
```

Comment:
I wonder if we just provide a render slot for anything to take up the space that the inputButton would. Since this abstraction is becoming more and more hairy. That would make this abstraction more flexible?

Reply:
done and done ✅